README.md

```diff
@@ -36,11 +36,16 @@ dependencies {
 }  
 ```
 3. Build your project. The plugin will generate json files for each Dagger component.
-4. Checkout out the `dagger-browser` project
+
+4. Use [scripts/mkmanifest.sh](https://github.com/Snapchat/dagger-browser/blob/master/scripts/mkmanifest.sh) to aggregate the component json files into a ComponentsManifest.json.
+
+5. Open a [Dagger Browser instance]((https://snapchat.github.io/dagger-browser/plaid)), and drag-and-drop your ComponentsManifest.json file to load it.
```

Comment:
change this to https://snapchat.github.io/dagger-browser/home/#/

which is the standalone version with nothing loaded.